### PR TITLE
 Implement `attach --create` subcommand flag to create session if one does not exist

### PR DESCRIPTION
**Pull request by GPery**
Hey!
Addressing #541 to help my automated zellij remote workspace :D

Thanks!

### CODE REVIEW

File: zellij-utils/src/cli.rs [hunk 1]
```diff
@@ -85,6 +85,11 @@ pub enum Sessions {
         /// zellij client (if any) and attach to this.
         #[structopt(long, short)]
         force: bool,
+
+        /// Create a session if one does not exist.
+        #[structopt(long)]
```
Comment (kunalmohan) [inline zellij-utils/src/cli.rs:90]: Let's have the `short` version as well.

File: src/sessions.rs [hunk 2]
```diff
@@ -1,12 +1,13 @@
 use std::os::unix::fs::FileTypeExt;
 use std::{fs, io, process};
 use zellij_utils::{
+    async_std::io::ErrorKind,
```
Comment (kunalmohan) [inline src/sessions.rs:4]: Why can't we use `std::io::ErrorKind` instead of this?

**Review by kunalmohan — CHANGES_REQUESTED** [review]
@GPery Thanks for taking this up! This looks very good. Just a couple of small changes and it should be ready to merge.

File: src/sessions.rs (hunk 2 above)
Comment (GPery) [reply src/sessions.rs:4]: No reason, fixed

File: zellij-utils/src/cli.rs (hunk 1 above)
Comment (GPery) [reply zellij-utils/src/cli.rs:90]: Of course
I initially removed the short option because it was for `zellij` and not for `attach`
Added

**Review by kunalmohan — APPROVED** [review]
LGTM! Thanks for working on this! It is a much-needed feature.

**Pull request merged**